src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -83,49 +86,6 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
         {
         }
 
-        // 15.6 BACKCOMPAT OVERLOAD -- DO NOT TOUCH
```

Comment:
i am worried that you removed the thing that says DO NOT TOUCH :)

Reply:
ah, you added below.  ok

Reply:
Yeah just moved down with the rest. It was in the wrong place.

Reply:
> Yeah just moved down with the rest. 

Do you think that makes this change easier or harder to review?

> It was in the wrong place.

Why do you think it was in the wrong place. What is the definition of right place. Is there a specific guideline violated?  



---
In reply to: [177905162](https://github.com/dotnet/roslyn/pull/25796#discussion_r177905162) [](ancestors = 177905162)